presto-main/src/test/java/io/prestosql/tests/ReportIllNamedTest.java

```diff
@@ -0,0 +1,30 @@
+package io.prestosql.tests;
+
+import org.testng.IClassListener;
+import org.testng.ITestClass;
+
+import static java.lang.String.format;
+
+public class ReportIllNamedTest
```

Comment:
`Ill` looks like an abbreviation. How about `Incorrectly`

Reply:
This is for "ill-named." I don't think it matters anyway. I will keep as is